src/handlers/poster-gewis/index.tsx

```diff
@@ -0,0 +1,35 @@
+import PosterBaseView from '../poster-base';
+import { PhotoPoster as ClientPhotoPoster, Poster } from '../../api';
+import OverlayGewis from './components/OverlayGewis';
+import BorrelLogoPoster from './components/types/BorrelLogoPoster';
+import BorrelWallOfShamePoster from './components/types/BorrelWallOfShame';
+import BorrelPriceListPoster from './components/types/BorrelPriceListPoster';
+import TrainPoster from './components/types/TrainPoster';
+import OlympicsPoster from './components/types/OlympicsPoster';
+import PhotoPoster from './components/types/PhotoPoster';
+
+export default function PosterGewisView() {
+  const localPosterRenderer = (poster: Poster, visible: boolean, setTitle?: (title: string) => void) => {
+    // TODO: Properly handle this
+    if (setTitle == undefined) return <div>setTitle not defined</div>;
```

Comment:
can you the not make it required?

Reply:
Yes, good point, thought there was a reason this was not possible, but there was not, fixed!